modules/02-interactive-viz/index.md

```diff
@@ -10,17 +10,956 @@ authors:
 
 # 👁️ 2 - Interactive visualization of raster and vector data
 
-:::{note} 🛝 Slides
-:icon: false
-:class: dropdown
-
-<iframe
-    src="https://docs.google.com/presentation/d/e/2PACX-1vRPPScGGDg9PKQiEQGCj5GWSt5TnrHUyTOPoSRKrLAs9sXKuj2BTwc4SzCpXYrDE2p6B7hpRFgpoguC/pubembed?start=false&loop=false&delayms=60000"
-    frameborder="0"
-    width="960"
-    height="569"
-    allowfullscreen="true"
-    mozallowfullscreen="true"
-    webkitallowfullscreen="true">
-</iframe>
-:::
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/opengeos/leafmap/blob/master/docs/workshops/AGU_2025.ipynb)
+
+## Introduction
+
+This notebook is for the workshop ([Open Source Geospatial Workflows in the Cloud](https://geojupyter.github.io/workshop-open-source-geospatial)) presented at the [AGU Fall Meeting 2025](https://agu.confex.com/agu/agu25/meetingapp.cgi/Session/252640).
+
+### Learning Objectives
+
+By the end of this workshop, you will be able to:
+
+- Create interactive maps with MapLibre using Python
+- Add and customize various data layers (raster, vector, 3D)
+- Visualize remote sensing data including COGs, STAC, and local rasters
+- Work with PMTiles for efficient vector tile rendering
+- Create 3D visualizations including terrain, buildings, and indoor maps
+- Use TiTiler for dynamic raster tile serving
+- Apply custom styling and legends to enhance map readability
+
+## Useful Resources
+
+- [MapLibre GL JS Documentation](https://maplibre.org/maplibre-gl-js/docs): Comprehensive documentation for MapLibre GL JS.
+- [MapLibre Python Bindings](https://github.com/eoda-dev/py-maplibregl): Information on using MapLibre with Python.
+- [MapLibre in Leafmap](https://leafmap.org/maplibre/overview): Examples and tutorials for MapLibre in Leafmap.
+- [Video Tutorials](https://bit.ly/maplibre): Video guides for practical MapLibre skills.
+- [MapLibre Demos](https://maps.gishub.org): Interactive demos showcasing MapLibre's capabilities.
+
+## Table of Contents
```

Comment:
A table of contents will automatically be rendered on the right sidebar. Do you feel we can remove this ToC and use the sidebar one?